tests/test_laser_profiles.py

```diff
@@ -234,6 +238,89 @@ def test_longitudinal_profiles():
     print("cep_phase = ", cep_phase_cos)
     assert np.abs(cep_phase_cos - cep_phase) / cep_phase < 0.02
 
+    # LongitudinalProfileFromData
+    print("LongitudinalProfileFromData")
+    data = {}  # Generate spectral data assuming analytic ft of GaussianLongitudinalProfile
```

Comment:
Is this correct? Just to make it more explicit.
```suggestion
    data = {}  # Generate spectral data assuming analytic Fourier transform of GaussianLongitudinalProfile
```